CHANGELOG.md

```diff
@@ -1,7 +1,14 @@
 # Change Log
 All notable changes to this project will be documented in this file. This project adheres to [Semantic Versioning](http://semver.org)
 
-## 3.1.2-dev
+## 3.1.2 - 2022-12-16
```

Comment:
```suggestion
## 3.1.2 - 2022-12-15
```
Are we releasing 3.1.2 today?

Reply:
I was planning on doing it tomorrow but may be postponed as per recent Slack conversation

Reply:
Either way, this LGTM 👍